Doc/library/wsgiref.rst

```diff
@@ -754,6 +754,49 @@ input, output, and error streams.
    .. versionadded:: 3.2
 
 
+:mod:`wsgiref.types` -- WSGI types for static type checking
```

Comment:
We can also add something to the intro for the wsgiref package, in the paragraph that starts "[wsgiref](https://docs.python.org/3.10/library/wsgiref.html#module-wsgiref) is a reference implementation...". Similarly, we can link the remaining documentation to `wsgiref.types`, for example in the sentence "All of the functions taking an environ parameter expect a WSGI-compliant dictionary to be supplied",